formats/properties/commonMain/src/kotlinx/serialization/Properties.kt

```diff
@@ -10,7 +10,7 @@ import kotlinx.serialization.modules.*
 
 /**
  * Transforms a [Serializable] class' properties into a single flat [Map] which consists of
- * string keys and primitive type values, and vice versa.
+ * string keys and primitive type values, and vice versa. Located in separated `kotlinx-serialization-properties` artifact.
```

Comment:
Did you add this mention for the website users?

Reply:
Yes